pkg/operations/kubernetesupgrade/upgrader.go

```diff
@@ -363,7 +363,7 @@ func (ku *Upgrader) upgradeAgentPools(ctx context.Context) error {
 			ku.logger.Infof("Upgrading Agent VM: %s, pool name: %s", vm.name, *agentPool.Name)
 
 			// copy custom properties from old node to new node if the PreserveNodesProperties in AgentPoolProfile is not set to false explicitly.
-			preserveNodesProperties := false
+			preserveNodesProperties := true
```

Comment:
Should we capture this default setting somewhere in the default.go? 
@CecileRobertMichon

Reply:
Yes definitely. Let's add a constant for the default as well.

Reply:
Add default setting.